src/Privacy.ts

```diff
@@ -3,60 +3,4 @@ import type { PrivacyType } from './types/Privacy';
 
 const { AppLovinMAX } = NativeModules;
 
-/**
- * This enum represents the user's geography used to determine the type of consent flow shown to the
- * user.
- */
-export enum ConsentFlowUserGeography {
-    /**
-     * User's geography is unknown.
-     */
-    UNKNOWN = 'U',
-
-    /**
-     * The user is in GDPR region.
-     */
-    GDPR = 'G',
-
-    /**
-     * The user is in a non-GDPR region.
-     */
-    OTHER = 'O',
-}
-
-/**
- * AppLovin SDK-defined app tracking transparency status values (extended to include "unavailable"
- * state on iOS before iOS14).
- */
-export enum AppTrackingStatus {
-    /**
-     * Device is on iOS before iOS14, AppTrackingTransparency.framework is not available.
-     */
-    UNAVAILABLE = 'U',
-
-    /**
-     * The user has not yet received an authorization request to authorize access to app-related
-     * data that can be used for tracking the user or the device.
-     */
-    NOT_DETERMINED = 'N',
-
-    /**
-     * Authorization to access app-related data that can be used for tracking the user or the device
-     * is restricted.
-     */
-    RESTRICTED = 'R',
-
-    /**
-     * The user denies authorization to access app-related data that can be used for tracking the
-     * user or the device.
-     */
-    DENIED = 'D',
-
-    /**
-     * The user authorizes access to app-related data that can be used for tracking the user or the
-     * device.
-     */
-    AUTHORIZED = 'A',
-}
-
 export const Privacy: PrivacyType = AppLovinMAX;
```

Comment:
Any reason to still keep this file?

Reply:
We still have the functions defined in [src/types/Privacy.ts](https://github.com/AppLovin/AppLovin-MAX-React-Native/pull/300/files#diff-43edab956f944f89cd8767798de23c6fdd54fc32bfa3ac06ff346d3af96cab63).   Do we want to move all of them to `AppLovinMAX`?

Reply:
No we don't, that would break a ton of integrations. Can we somehow combine the files? Since this file doesn't have any APIs declared anymore

Reply:
The `Privacy` module is defined here by filtering `AppLovinMAX`, which is actually the implementation for JavaScript.    But, the API declaration in [src/types/Privacy.ts](https://github.com/AppLovin/AppLovin-MAX-React-Native/pull/300/files#diff-43edab956f944f89cd8767798de23c6fdd54fc32bfa3ac06ff346d3af96cab63) means nothing for JavaScipt but the main things for TypeScript.   These two files should not be combined to make a clear distinction between `type` and implementation.   Also, when compiling with JavaScript,  all types in the `type` directory and the imported modules with `type` will be ignored.